### PR TITLE
Api that returns archived tasks

### PR DESCRIPTION
**Pull request by timiajayi**
added archive and archived

### CODE REVIEW

File: app/Http/Controllers/TaskController.php [hunk 1]
```diff
@@ -158,4 +158,17 @@ public function showResource(Request $request) : TodoResourceCollection
         $tasks = $this->taskService->all();
         return new TodoResourceCollection($tasks);
     }
+    public function archived()
```
Comment (kayleb01) [inline app/Http/Controllers/TaskController.php:161]: This is not how to go about doing this. We are not using eloquent or anything like that
so there's no static where method you can call on the task

File: app/Http/Controllers/TaskController.php [hunk 2]
```diff
@@ -158,4 +158,17 @@ public function showResource(Request $request) : TodoResourceCollection
...
+    {
+        $tasks = $this->taskService->all();
+        return $tasks::where('archive', 1)
+                       ->orderBy('id', 'desc')->get();
+    }
+    public function archive($id)
+    {
+        $tasks = $this->taskService->all();
```
Comment (kayleb01) [inline app/Http/Controllers/TaskController.php:169]: And there's no find method either

File: app/Http/Controllers/TaskController.php (hunk 1 above)
Comment (timiajayi) [reply app/Http/Controllers/TaskController.php:161]: I’m not the one that did todo resourcecollection o

Comment (timiajayi) [reply app/Http/Controllers/TaskController.php:161]: Can I use soft delete?

**Comment by kayleb01** [comment]
I have commented on your code, please check and fix accordingly

**Pull request closed without merging**